plugins/modules/zos_copy.py

```diff
@@ -2007,7 +2014,8 @@ def main():
             copy_member=dict(type='bool'),
             src_member=dict(type='bool'),
             local_charset=dict(type='str'),
-            force=dict(type='bool', default=False)
+            force=dict(type='bool', default=False),
+            tmphlq=dict(type='str', required=False, default=None)
```

Comment:
There is confusion... what I thought I saw was 'tmphlq' is a string/parameter, and tmp_hlq is a dictionary.  As I look more closely, I'm seeing them intermixed.  I'll try to put comment notes where I see tmphlq as a dictionary.